packages/fast-components-react-base/src/auto-suggest/auto-suggest.tsx

```diff
@@ -63,6 +68,8 @@ class AutoSuggest extends Foundation<
 
     private shouldFocusMenuOnNextRender: boolean = false;
 
+    private storedSearchString: string;
```

Comment:
I think this should be `storedValueString` since AutoSuggest may not be used for searching